README.md

```diff
@@ -1 +1,68 @@
-# Awesome_Books
\ No newline at end of file
+# Awesome_Books
+
+> In this project we created a simple book app wich can store and remove a specific book
+from the list usig objects and classes. The localstorage was also used to store with safety the user's 
+data
+
+## How it looks like :
+
+![AwesomeBook2](https://user-images.githubusercontent.com/94958024/159319890-f97fac56-4400-46fd-ab8b-47f8271b52a9.png)
+## Built With
+
+- HTML
+- JavaScript
+
+## Live Demo is avaible here
+
+[Live Demo Link](https://anicetfantomas.github.io/Awesome_Books/)
+
+
+## Getting Started
+
+```
+To get a local copy up and running follow these simple example steps.
+
+git clone 
+https://github.com/AnicetFantomas/Awesome_Books.git
+
+```
+
+### Deployment
+
+https://anicetfantomas.github.io/Awesome_Books/
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/AnicetFantomas)
+
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/anicet-murhula-13a1b0220/)
+
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/AnicetFantomas)
+
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/anicet-murhula-13a1b0220/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] The MIT link is broken. I would recommend fixing this so your README is documented professionally.